src/arrayMethodJoin.js

```diff
@@ -1,11 +1,41 @@
 'use strict';
 
+const nullString = 'null';
+const emptyString = '';
```

Comment:
Keep rules of constant naming
```suggestion
const NULL_STRING= 'null';
const EMPTY_STRING= '';
```